runtime/doc/diagnostic.txt

```diff
@@ -340,6 +340,20 @@ Example: >lua
       end,
     })
 <
+==============================================================================
+MAPS                                                    *diagnostic-maps*
+
+                                                       *]d-default*
+]d                        Goto the next diagnostic in the buffer.
+                            |vim.diagnostic.goto_next()|
+                                                       *[d-default*
+[d                        Goto the previous diagnostic in the buffer.
+                            |vim.diagnostic.goto_prev()|
```

Comment:
Since these are relative to the current buffer, this seems fine. For iterating the *global* diagnostics quickfix list, I'd suggest `]q` and `[q` (mnemonic: quickfix list), which was established by https://github.com/tpope/vim-unimpaired/ and has harmony with other quickfix-related potential mappings.